gulpfile.js

```diff
@@ -117,9 +117,24 @@ const appBundles = [{
   transforms: ['babel', 'coffee'],
 }];
 
-const appBundleConfigs = appBundles.map(function (config) {
-  return Object.assign({}, appBundleBaseConfig, config);
-});
+const polyfillBundles = [
+  'document.evaluate',
+  'es2015',
+  'es2016',
+  'es2017',
+  'string.prototype.normalize',
+  'url',
+].map(set => ({
+  name: `polyfills-${set}`,
+  entry: `./src/shared/polyfills/${set}`,
+  transforms: ['babel'],
+}));
+
+const appBundleConfigs = appBundles
+  .concat(polyfillBundles)
+  .map(config => {
+    return Object.assign({}, appBundleBaseConfig, config);
+  });
```

Comment:
This creates a new `polyfills-{set}.bundle.js` JS bundle per group of polyfills. The polyfill sets are grouped by JS version, with larger polyfills and DOM polyfills separated out into their own bundles.